### PR TITLE
EID-2080 acceptance tests for no eIDAS

### PR DESCRIPTION
**Pull request by JohnWattsGDS**
Remove acceptance test steps that move a journey through the "journey picker" page, that lets users choose to use either Verify or eIDAS.

Tag smoke tests that send a user through the country picker page as `@Eidas` so that they can be ignored later in integration and production.

Tested against a local Hub running without eidas.

### CODE REVIEW

File: pre-commit.sh [hunk 1]
```diff
@@ -13,10 +13,10 @@ if [[ ! -z $1 ]]
 then
   bundle --quiet
   mkdir -p testreport
-  SHOW_BROWSER=${SHOW_BROWSER:-false} TEST_ENV=${TEST_ENV:-local} bundle exec cucumber $* --strict --tags 'not @ignore'
+  SHOW_BROWSER=${SHOW_BROWSER:-false} TEST_ENV=${TEST_ENV:-local} bundle exec cucumber $* --strict --tags 'not (@Eidas or @staging-only)'
```
Comment (instantiator) [inline pre-commit.sh:16]: Should we keep the ability to ignore tests marked `@ignore`, too?

File: pre-commit.sh [hunk 2]
```diff
@@ -13,10 +13,10 @@ if [[ ! -z $1 ]]
...
   exit $?
 fi
 
 bundle --quiet
 mkdir -p testreport
-SHOW_BROWSER=${SHOW_BROWSER:-false} TEST_ENV=${TEST_ENV:-local} bundle exec parallel_cucumber features/ -n ${INSTANCES:-3} -o "--strict --tags 'not @ignore'"
+SHOW_BROWSER=${SHOW_BROWSER:-false} TEST_ENV=${TEST_ENV:-local} bundle exec parallel_cucumber features/ -n ${INSTANCES:-3} -o "--strict --tags 'not (@Eidas or @staging-only)'"
```
Comment (instantiator) [inline pre-commit.sh:22]: Same question :)

File: run-tests-with-docker.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ sleep $((2+NODES))
 docker-compose run \
                --name verify-tests \
                -e TEST_ENV="${TEST_ENV:-"staging"}" \
-               verify-tests -n $NODES -o "--strict -f pretty -f junit -o testreport/ --tags 'not @ignore' $*"
+               verify-tests -n $NODES -o "--strict -f pretty -f junit -o testreport/ --tags 'not @Eidas' $*"
```
Comment (instantiator) [inline run-tests-with-docker.sh:13]: As in `pre-commit.sh` should we keep the ability to ignore tests marked `@ignore` too?

**Review by instantiator — APPROVED** [review]

File: pre-commit.sh (hunk 1 above)
Comment (idrop) [reply pre-commit.sh:16]: Good quesion @instantiator . I should have mentioned in the PR that the only tests tagged @ignore are eidas tests, so we want to tag them as eidas explicity

**Pull request merged**